csfieldguide/templates/chapters/chapter_section.html

```diff
@@ -29,16 +29,16 @@
   <div>
     {% if previous_section %}
       <a href="{% url 'chapters:chapter_section' chapter.slug previous_section.0.slug %}" class="btn btn-primary float-left mb-2 {% if not chapter_section.translation_available %} text-muted{% endif %}">
-        Previous: {{ previous_section.0.name }}
+        {% trans 'Previous:' %} {{ previous_section.0.name }}
```

Comment:
I think we also need to include `{{ previous_section.0.name }}` inside the trans tag if it allows you? I think the translators will understand what the variable means. The order may be different in different languages. This could also relate to your 'help-wanted' question?

If it breaks when you wrap it then ignore me :smiley_cat:

Reply:
It does break. I don't think it's an issue because it's next-*colon*-chaptername, so the order shouldn't change

Reply:
Does this link help at all? https://stackoverflow.com/questions/6546933/django-translate-variable-content-in-template

Reply:
It does not

Reply:
:crying_cat_face:

Reply:
We may need to summon @JackMorganNZ